src/shared_modules/router/src/remoteSubscriber.hpp

```diff
@@ -57,6 +57,7 @@ class RemoteSubscriber final
             {
                 if (!m_isRegistered)
                 {
+                    // LCOV_EXCL_START
```

Comment:
Question: Why is not possible to test this code?

Reply:
It's because the class publisher always has the same [response](https://github.com/wazuh/wazuh/blob/d73e3a093dc2113f426e7968a2bd22d651e3ed7d/src/shared_modules/router/src/publisher.hpp#L71), and I couldn't find any scenario in which to reproduce these cases
I can remove `LCOV_EXCL_START` and `LCOV_EXCL_STOP`, if you'd like

Reply:
Excelent